torch/csrc/generic/SparseTensor.cpp

```diff
@@ -108,12 +127,12 @@ static PyObject * THSPTensor_(pynew)(PyTypeObject *type, PyObject *args, PyObjec
   // torch.SparseTensor(size)
   if (num_args == 1 && THPUtils_checkLong(first_arg)) {
     long size = THPUtils_unpackLong(first_arg);
-    self->cdata = THSTensor_(newWithSize1d)(LIBRARY_STATE size);
+    self->cdata = THSTensor_(_newWithSize1d)(size);
   }
   // torch.SparseTensor(torch.Size sizes)
   else if (num_args == 1 && THPSize_Check(first_arg)) {
     THLongStoragePtr sizes(THPUtils_unpackSize(first_arg));
-    self->cdata = THSTensor_(newWithSize)(LIBRARY_STATE sizes.get());
+    self->cdata = THSTensor_(_newWithSize)(sizes.get());
```

Comment:
are these functions necessary? There's another call to `initStorage` below

Reply:
Ah I see! I assumed that this file was following `Tensor.cpp`. Updating now.